generator.yaml

```diff
@@ -1,6 +1,25 @@
 ignore:
   resource_names:
-    - Function
+    # Function
     - Alias
     - CodeSigningConfig
     - EventSourceMapping
+  shape_names:
+    - LayerList
```

Comment:
any reason to ignore this shape? I see that there is a `Status.Layers` field so where does this LayerList shape come into play?

Reply:
In the API schema description `LayerList` is a list of strings and not `[]*Layer`. I will address this issue in a future PR